group-coordinator/src/main/resources/common/message/ConsumerGroupPartitionMetadataValue.json

```diff
@@ -29,7 +29,27 @@
       { "name": "TopicName", "versions": "0+", "type": "string",
         "about": "The topic name." },
       { "name": "NumPartitions", "versions": "0+", "type": "int32",
```

Comment:
Do we still need this field? I suppose that it depends on whether you plan to store partition metadata for partitions without racks or not.

Reply:
I think we shouldn't store the racks if they aren't available, I just changed it to remove numPartitions but I'll change it back now